projects/components/src/datagrid/datagrid.component.spec.ts

```diff
@@ -296,6 +296,14 @@ describe('DatagridComponent', () => {
                     this.clrGridWidget.getSelectionLabelForRow(1).click();
                     await timeout();
                     expect(this.hostComponent.selectionChanged).toHaveBeenCalledWith(mockData);
+                    await timeout();
+                    this.finder.detectChanges();
+                    await timeout();
+                    this.finder.detectChanges();
+                    await timeout();
+                    this.finder.detectChanges();
+                    await timeout();
+                    expect(this.hostComponent.selectionChanged).toHaveBeenCalledTimes(3);
```

Comment:
Shouldn't this verify that it has been called just once? Why `selectionChanged` should have been called all the time provided that there is no activity that leads to a real change?

Reply:
I think this is the correct behavior, but it's unclear because I tacked this onto an existing test, which was a bad decision. Moving it to a new `it`